cfgov/v1/util/util.py

```diff
@@ -68,6 +68,9 @@ def get_secondary_nav_items(request, current_page):
     else:
         page = get_appropriate_page_version(request, current_page)
 
+    if not page:
```

Comment:
I don't know whether this has ever worked to preview new BrowsePages, although it may work when previewing edited ones. In the code there's a reference to `current_page.get_parent()` which will always be `None` when previewing a new page.